nfs-ganesha/build/build_deb

```diff
@@ -36,15 +44,24 @@ cd build
 # make source tarball
 cmake -DCMAKE_BUILD_TYPE=Maintainer -DUSE_FSAL_GLUSTER=OFF -DUSE_FSAL_CEPH=ON -DUSE_FSAL_RGW=ON $WORKSPACE/nfs-ganesha/src && make dist || exit 1
```

Comment:
I recommend checking with Kaleb Keithley to see how he creates the tarball for Debian packages [in the PPA](https://launchpad.net/~gluster/+archive/ubuntu/nfs-ganesha), because he must've dealt with this issue.